recipes/tethys-platform/build.sh

```diff
@@ -0,0 +1 @@
+PBR_VERSION=3.4.3 $PYTHON -m pip install . -vv
```

Comment:
```suggestion
PBR_VERSION=%PKG_VERSION% $PYTHON -m pip install . -vv
```
https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process